src/js/components/ChromecastButton.js

```diff
@@ -43,6 +43,16 @@ ChromecastButton = {
       } else {
          this._onChromecastDevicesUnavailable();
       }
+
+      if (options.addCastLabelToButton) {
+         this.el().classList.add('vjs-chromecast-button-lg');
```

Comment:
When the button has a label, we should remove the control text, otherwise screen readers will read both labels, one right after the other.